modules/back-end/src/Infrastructure/Users/UserService.cs

```diff
@@ -22,12 +22,12 @@ public async Task<string> GetOperatorAsync(Guid operatorId)
         var user = await FindOneAsync(x => x.Id == operatorId);
         if (user is not null)
         {
-            return user.Name ?? user.Email;
+            return !string.IsNullOrWhiteSpace(user.Name) ? user.Name : user.Email;
         }
 
         // An operation can also be made by an access token through our Open Api, see "OpenApiHandler"
         var accessToken = await MongoDb.QueryableOf<AccessToken>().FirstOrDefaultAsync(x => x.Id == operatorId);
-        return accessToken?.Name ?? string.Empty;
+        return !string.IsNullOrWhiteSpace(accessToken.Name) ? accessToken.Name : string.Empty;
```

Comment:
This change seems unnecessary

Reply:
Ah, good catch! I wasn't paying close enough attention to recognize that this is correctly handling the empty string. Will fix this shortly.